### ISSUE TITLE
tracking detector

### ISSUE THREAD

**Issue by fritzbee** [studio-minus/ppg-bugs-and-requests#2264]
**Feature description**
it's like the turrets except they send an activation signal

**Feature purpose**
it's purpose is to create customisable turrets and alike

**Comment by DindinYT37** [comment]
Duplicate of or Related to:
* #278
* #1007
* #1048
* #1099
* #1911 

*I'm also pretty sure this was already requested under the name of "Enemy core", though I am unable to find that.*

**Comment by fritzbee** [comment]
i wish i knew how to use the automatic sentry controller thingy

**Comment by DindinYT37** [comment]
> i wish i knew how to use the automatic sentry controller thingy

That's actually part of the [Modding API ](https://www.studiominus.nl/ppg-modding/), it's to allow mods to create new sentries using it.

**Comment by fritzbee** [comment]
> > i wish i knew how to use the automatic sentry controller thingy
> 
> That's actually part of the [Modding API](https://www.studiominus.nl/ppg-modding/), it's to allow mods to create new sentries using it.

yeah from what I see it requires some C# knowledge and i don't have a single clue about how to program in C#

**Comment by DindinYT37** [comment]
> > > i wish i knew how to use the automatic sentry controller thingy
> > 
> > 
> > That's actually part of the [Modding API](https://www.studiominus.nl/ppg-modding/), it's to allow mods to create new sentries using it.
> 
> yeah from what I see it requires some C# knowledge and i don't have a single clue about how to program in C#

It's never too late to learn a skill, but even if you don't know – there'll always be someone in the community that would do so if you asked them.

**Comment by royalealbert27** [comment]
so.. flying detectors???

**Comment by bibbybabby** [comment]
> so.. just detectors?

detectors that look at humans, like turrets do
